EventListener/Login.php

```diff
@@ -63,6 +67,21 @@ public function setCustomerId($value)
         $this->options['customerId'] = $value;
     }
 
+    protected function getUserIdentifier(TokenInterface $token)
+    {
+        $tokenUser = $token->getUser();
+
+        if ($tokenUser instanceof eZUser || !$tokenUser instanceof UserInterface) {
+            return $tokenUser->getAPIUser()->id;
+        }
+
+        if (method_exists($tokenUser, 'getId')) {
+            return $tokenUser->getId();
+        }
+
+        return $tokenUser->getUsername();
+    }
+
     public function onSecurityInteractiveLogin(InteractiveLoginEvent $event)
     {
         if ($this->authorizationChecker->isGranted('IS_AUTHENTICATED_FULLY') | // user has just logged in
```

Comment:
Probably rhetorical question, but it was you @clash82 9f757d71672e4563e214a44d4b6cc4afd020af06 ;)